src/AlgoliaEngine.php

```diff
@@ -1,35 +1,34 @@
 <?php
 
-namespace Algolia\SearchBundle\Engine;
+namespace Algolia\SearchBundle;
 
-use Algolia\SearchBundle\SearchableEntityInterface;
-use AlgoliaSearch\Client;
+use Algolia\AlgoliaSearch\SearchClient as Client;
 
-class AlgoliaEngine implements EngineInterface
+class AlgoliaEngine
 {
-    /** @var Client Client */
+    /** @var Client */
     protected $algolia;
 
     public function __construct(Client $algolia)
     {
         $this->algolia = $algolia;
     }
 
-    public function add($searchableEntities)
+    public function add($searchableEntities, $requestOptions = [])
```

Comment:
Don't forget to test that all those new `requestOptions` work as expected.

Reply:
done